src/angularjs/src/app/components/analysis-jobs/analysis-jobs-status.filter.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @ngdoc filter
+ * @name pfb.analysis-jobs.status:displayStatus
+ *
+ * @description
+ * Transforms analysis job status into user friendly string
+ */
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function displayStatus() {
+        var statuses = {
+            CREATED: 'Created',
```

Comment:
These are defined both here and in analysis-jobs.service.js. Consider defining in a constant service and reusing them here.